src/Database/NeoServer.Data/Seeds/PlayerModelSeed.cs

```diff
@@ -177,9 +177,9 @@ public static void Seed(EntityTypeBuilder<PlayerModel> builder)
             {
                 PlayerId = 5,
                 PlayerType = 1,
-                AccountId = 1,
+                AccountId = 2,
                 TownId = 1,
-                Name = "Paladin Sample",
+                Name = "Paladin",
```

Comment:
why removing sample from Paladin but not from other vocations? ie : Sorcerer Sample...

Reply:
I wanted add multiple account.